session.go

```diff
@@ -603,15 +624,23 @@ func (s *Session) mux(remoteBegin *frames.PerformBegin) {
 				delete(handlesByDeliveryID, deliveryID)
 			}
 
+			s.txFrame(env.Ctx, fr, env.Sent)
+			if sendErr := <-env.Sent; sendErr != nil {
```

Comment:
`env.Sent` can be nil, correct? Or not in this particular spot?

Reply:
I can be `nil`, but for sending transfer frames it never will be (I will add a comment to `transferEnvelope` explaining this).